### ISSUE TITLE
Lose visual focus when navigating radio buttons (Safari only)

### ISSUE THREAD

**Issue by terracoda** [phetsims/balloons-and-static-electricity#321]
I loose keyboard focus when trying to navigate beyond the last radio buttons. This happen when I press either the down-arrow key or the right-arrow key on the last radio. It does not happen on the first radio when I press the up- or left-arrow keys.

This happen only in Safari and with or without VO on.

I think the pressing the down- or right-arrow on the last radio should cycle either do nothing, or cycle back to top of radio list as it does in other browsers (see other issue on this - will add).

When visual focus is lost, the cursor focus moves to the top of the page to the alert content for the selected radio.

Note added: Used current build [Oct. 25](http://phettest.colorado.edu/balloons-and-static-electricity/balloons-and-static-electricity_en.html?ea&brand=phet&accessibility)

Safari 10.0.3
Sierra: 10.12.3

**Comment by terracoda** [comment]
@jessegreenberg, this issue may be related to the issue in closed issue in MAL repo https://github.com/phetsims/molecules-and-light/issues/153.

**Comment by jessegreenberg** [comment]
Thanks @terracoda , I will take a look next time I have access to safari.

**Comment by jessegreenberg** [comment]
It is pretty likely that this was fixed by a commit in https://github.com/phetsims/balloons-and-static-electricity/issues/322. Ill check.

**Comment by jessegreenberg** [comment]
Reproduced bug, not fixed by #322.

**Comment by jessegreenberg** [comment]
It is not specific to radio buttons, happens for other buttons too. If I press the 'down' arrow after pressing the remove wall button, the arrow keys read through the aria-live content at the very top of the page. Happens most consistently for radio button group.

But it doesn't ready any farther, at the end of the alert, VO says "End of Text".

**Comment by jessegreenberg** [comment]
This seems to be fixed by clearing the alert content once it has been spoken so that it cannot be found with the virtual cursor. This has been desired for a while anyway, so I am going to commit it after testing to make sure that things still sound good with NVDA and JAWS.

**Comment by jessegreenberg** [comment]
Everything seems great in FF 56 with NVDA.

**Comment by jessegreenberg** [comment]
Sounds great with JAWS + Firefox 56.

**Comment by jessegreenberg** [comment]
Fixed with the above commit. Closing.

**Comment by jessegreenberg** [comment]
And thanks for finding @terracoda!
